instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/datasource/OpenTelemetryDataSource.java

```diff
@@ -40,29 +42,41 @@
 public class OpenTelemetryDataSource implements DataSource, AutoCloseable {
 
   private final DataSource delegate;
+  private final OpenTelemetry openTelemetry;
+
+  /**
+   * Create a OpenTelemetry DataSource wrapping another DataSource.
+   *
+   * @param delegate the DataSource to wrap
+   */
+  public OpenTelemetryDataSource(DataSource delegate) {
```

Comment:
We should deprecate this variant.

Reply:
Just to clarify by deprecate did you mean to remove or to mark as deprecated?

Reply:
Mark as deprecated - let's give our users some time to migrate.